Lib/test/test_doctest.py

```diff
@@ -2659,12 +2663,52 @@ def test_testfile(): r"""
     >>> sys.argv = save_argv
 """
 
+class TestLoader(importlib.abc.MetaPathFinder, importlib.abc.ResourceLoader):
```

Comment:
This is misnamed as this is implementing a finder and a loader, so call it an importer.

Reply:
I'll push an update changing the name to TestImporter.

Reply:
Commit bb836d7 does the name change.